Google.Ads.GoogleAds/examples/AccountManagement/VerifyAdvertiserIdentity.cs

```diff
@@ -73,37 +76,49 @@ public void Run(GoogleAdsClient client, long customerId)
             IdentityVerification identityVerification =
                 GetIdentityVerification(client, customerId);
 
-            if (identityVerification != null)
-            {
-                if (identityVerification.VerificationProgress.ActionUrl == null)
-                {
-                    StartIdentityVerification(client, customerId);
-
-                    // Call GetIdentityVerification again to retrieve the verification progress
-                    // after starting an identity verification session.
-                    GetIdentityVerification(client, customerId);
-
-                } else {
-                    // If there is an identity verification session in progress, there is no need
-                    // to start another one by calling StartIdentityVerification.
-                    Console.WriteLine("There is an advertiser identity verification session in " +
-                        "progress.\n" +
-                        "The URL for the verification process is: " +
-                        identityVerification.VerificationProgress.ActionUrl +
-                        " and it will expire at " +
-                        identityVerification.VerificationProgress.InvitationLinkExpirationTime);
-                }
-            }
-            else
+            if (identityVerification == null)
             {
+
                 // If GetIdentityVerification returned an empty response, the account is not
                 // enrolled in mandatory identity verification.
                 Console.WriteLine($"Account {customerId} is not required to perform advertiser " +
                     "identity verification.\n" +
                     "See https://support.google.com/adspolicy/answer/9703665 for details on how " +
                     "and when an account is required to undergo the advertiser identity " +
                     "verification program.");
-
+                    return;
+            }
+            else
```

Comment:
Since you `return` above, you don't need to place the rest of this in an `else` block